components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/AuthenticatorAPI.java

```diff
@@ -167,9 +168,9 @@ public Response authenticate(@Context Request request, @FormDataParam ("username
 
     @POST
     @Produces (MediaType.APPLICATION_JSON)
-    @Path ("/logout")
-    public Response logout(@Context Request request) {
-        String appContext = AuthUtil.getAppContext(request);
+    @Path ("/logout/{appName}")
```

Comment:
I think this app is dcr app which name starts with publisher, store and admin. Hence I believe it won't cause any issues. If app name going to contains spaces and special characters we might need to revisit.